classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -61,6 +61,8 @@ public function register_routes() {
 	public function update_items( $request ) {
 		$request_body = $request->get_body();
 		$settings = json_decode( $request_body, true, WOOCOMMERCE_CONNECT_MAX_JSON_DECODE_DEPTH );
+		$this->settings_store->update_origin_address( $settings[ 'origin' ] );
```

Comment:
Why did you move the call to update the store's origin address to the purchase step?